### ISSUE TITLE
[BUG] Make helm_chart, helm_directory & manifest_directory mutually exclusive flags

### ISSUE THREAD

**Issue by denverwilliams** [cncf/cnf-conformance#547]
**Describe the bug**
Make helm_chart, helm_directory & manifest_directory mutually exclusive flags.

Which means the cnf-conformance.yml should only have one of these keys set in the configuration at a time, and the test suite should error if multiple are being used, including empty value keys.

**Comment by lixuna** [comment]
FYI - This update has been included in v1.10.0. If any issues are found during peer review, please open a new bug ticket.

**Comment by agentpoyo** [comment]
This checks out. Creating a cnf-conformance.yml with all empty lines for each of the values results in error:

```
$ LOG_LEVEL=error ./cnf-conformance cnf_setup cnf-config=./cnf-conformance.yml
Successfully created directories for cnf-conformance
Error: Must populate at lease one installation type in ./cnf-conformance.yml: choose either helm_chart, helm_directory, or manifest_directory in cnf-conformance.yml!
```

**Comment by agentpoyo** [comment]
Although this passes; this still displays crystal code error in the output when LOG_LEVEL=error

Crystal code should likely only display when in DEBUG.

```
src/tasks/utils/cnf_manager.cr:423:7 in 'cnf_installation_method'
src/tasks/utils/cnf_manager.cr:39:24 in 'parse_config_yml'
src/tasks/utils/cnf_manager.cr:692:5 in 'sample_setup'
src/tasks/cnf_setup.cr:110:3 in '->'
../../../.crenv/versions/0.35.1/share/crystal/src/primitives.cr:255:3 in 'call'
lib/sam/src/sam/execution.cr:20:7 in 'invoke'
lib/sam/src/sam.cr:35:5 in 'invoke'
lib/sam/src/sam.cr:53:7 in 'process_tasks'
src/cnf-conformance.cr:94:3 in '__crystal_main'
../../../.crenv/versions/0.35.1/share/crystal/src/crystal/main.cr:105:5 in 'main_user_code'
../../../.crenv/versions/0.35.1/share/crystal/src/crystal/main.cr:91:7 in 'main'
../../../.crenv/versions/0.35.1/share/crystal/src/crystal/main.cr:114:3 in 'main'
__libc_start_main
_start
???
Error: Must populate at lease one installation type in ./cnf-conformance.yml: choose either helm_chart, helm_directory, or manifest_directory in cnf-conformance.yml!
```

**Comment by agentpoyo** [comment]
![547_conformance_output](https://user-images.githubusercontent.com/14348948/105102896-3910ae00-5a75-11eb-8391-18b4871811c7.png)

**Comment by lixuna** [comment]
FYI - This enhancement has been included in v0.10.0. 

A related, new issue has been opened - #580
